curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60f8604682407e0d017bbf7f.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-25
 
 # --description--
 
-For the terms and conditions, add an `input` with a `type` of `checkbox` to the third `label` element. Also, as we do not want users to sign up, without having read the terms and conditions, make it `required`.
+For the terms and conditions, add an `input` with a `type` of `checkbox` to the third `label` element. Also, as users should not sign up, without having read the terms and conditions, make it `required`.
```

Comment:
I think this makes the sentence more straightforward
```suggestion
For the terms and conditions, add an `input` with a `type` of `checkbox` to the third `label` element. Make this `input` element `required` because users should not sign up without reading the terms and conditions.
```

Reply:
@jeremylt I agree your suggestion because It is smooth and readable. I'm not native English speaker, but I feel so.